curriculum/challenges/english/14-responsive-web-design-22/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-002.md

```diff
@@ -1,56 +1,26 @@
 ---
-id: 6193da99ca92051a07fb8a13
+id: 61fd6343fb42013d99bcd7f3
 title: Step 2
 challengeType: 0
 dashedName: step-2
 ---
 
 # --description--
 
-Within your `body` element, create a `div` element with the `id` set to `sheet`. Below that, create a `footer` element with the text set to `Last Updated: December 2021`.
+Within your `body` element, nest a `section` element within a `main` element.
 
 # --hints--
 
-You should create a new `div` element.
+You should have a `main` element within your `body` element.
```

Comment:
```suggestion
The first element inside the `body` element should be your `main` element. 
```

or something

as I got error with
```
  <body>
    <section></section>
    <main></main>
  </body>
```
and it can be confusing if one is not careful